tests/lib/rules/require-ember-lifeline.js

```diff
@@ -1,26 +1,31 @@
 const rule = require('../../../lib/rules/require-ember-lifeline');
-const MESSAGE = rule.meta.message;
+const getMessage = rule.meta.message;
 const RuleTester = require('eslint').RuleTester;
 const ruleTester = new RuleTester();
```

Comment:
Can we just pass the `parserOptions` object directly into the instantiation of `RuleTester()`. That way we can delete the redundant `parserOptions` from each assertion.